packages/management/tkr-service/bundle/config/upstream/tkr-crd.yaml

```diff
@@ -0,0 +1,383 @@
+            description: TanzuKubernetesReleaseStatus defines the observed state of
+              TanzuKubernetesRelease
+            properties:
+              conditions:
+                items:
+                  description: Condition defines an observation of a Cluster API resource
+                    operational state.
+                  properties:
+                    lastTransitionTime:
+                      description: Last time the condition transitioned from one status
+                        to another. This should be when the underlying condition changed.
+                        If that is not known, then using the time when the API field
+                        changed is acceptable.
+                      format: date-time
+                      type: string
+                    message:
+                      description: A human readable message indicating details about
+                        the transition. This field may be empty.
+                      type: string
+                    reason:
+                      description: The reason for the condition's last transition
+                        in CamelCase. The specific API may choose whether or not this
+                        field is considered a guaranteed API. This field may not be
+                        empty.
+                      type: string
+                    severity:
+                      description: Severity provides an explicit classification of
+                        Reason code, so the users or machines can immediately understand
+                        the current situation and act accordingly. The Severity field
+                        MUST be set only when Status=False.
+                      type: string
+                    status:
+                      description: Status of the condition, one of True, False, Unknown.
+                      type: string
+                    type:
+                      description: Type of condition in CamelCase or in foo.example.com/CamelCase.
+                        Many .condition.type values are consistent across resources
+                        like Available, but because arbitrary conditions can be useful
+                        (see .node.status.conditions), the ability to deconflict is
+                        important.
+                      type: string
+                  required:
+                  - lastTransitionTime
+                  - status
+                  - type
+                  type: object
+                type: array
+            type: object
+        type: object
+    served: true
+    storage: false
+    subresources:
+      status: {}
+  - additionalPrinterColumns:
+    - jsonPath: .spec.version
+      name: Version
+      type: string
+    - jsonPath: .status.conditions[?(@.type=='Compatible')].status
+      name: Compatible
+      type: string
+    - jsonPath: .metadata.creationTimestamp
+      name: Created
+      type: date
+    name: v1alpha3
```

Comment:
I see v1alpha2 version is missing in the CRD. We should include the v1alpha2 version as well. cc: @imikushin

Reply:
Addressed in 7643ba8dd64b622f5b99281a84910ad82310c6d2

I've simply copied over the v1alpha1 version and changed the version. I believe, the important thing to note was that the storage version remains false.